### PR TITLE
Fixed playground draggable block while scrolled

### PR DESCRIPTION
**Pull request by Legonois**
Last pull request did not include the fix for the actual on drop event, only for the placement of the target line. This adjusts the drop event to account for scrolling

### CODE REVIEW

**Comment by ivailop7** [comment]
I guess this PR is meant to supersede the #5839 ? So 5839 is closable?

**Comment by ivailop7** [comment]
Still not able to reproduce this issue in the current Playground

File: packages/lexical-playground/src/plugins/DraggableBlockPlugin/index.tsx [hunk 1]
```diff
@@ -361,7 +361,8 @@ function useDraggableBlockMenu(
         return true;
       }
       const targetBlockElemTop = targetBlockElem.getBoundingClientRect().top;
-      if (pageY / calculateZoomLevel(target) >= targetBlockElemTop) {
+      const adjsutedTop = targetBlockElemTop + window.scrollY;
+      if (pageY / calculateZoomLevel(target) >= adjsutedTop) {
```
Comment (potatowagon) [inline packages/lexical-playground/src/plugins/DraggableBlockPlugin/index.tsx:365]: nit: typo in `adjsutedTop`

Comment (ivailop7) [reply packages/lexical-playground/src/plugins/DraggableBlockPlugin/index.tsx:365]: @Legonois are you planning on fixing the typo, so we can proceed forward with the merge

**Comment by Legonois** [comment]
Just checked the deployments for the bug and it seems fixed! 

Tell me if there is anything else I need to do before this can be pulled to main

**Pull request closed without merging**